tlsconfig/config_test.go

```diff
@@ -240,11 +277,105 @@ func TestConfigServerTLSClientCASet(t *testing.T) {
 	if tlsConfig.ClientAuth != tls.VerifyClientCertIfGiven {
 		t.Fatal("ClientAuth was not set to what was in the options")
 	}
-	if tlsConfig.ClientCAs == nil || len(tlsConfig.ClientCAs.Subjects()) != 2 {
+	basePool, err := SystemCertPool()
```

Comment:
Note that this change is unrelated to this PR - these tests just didn't seem to pass on mac os previously.